### PR TITLE
Add support for console.clear

### PR DESCRIPTION
**Pull request by nchevobbe**
For issue #33

### CODE REVIEW

**Comment by nchevobbe** [comment]
Rebased (https://github.com/bgrins/gecko-dev/pull/82/commits/4e7bff19a4e3b5dcbf7e0f1fc100e1308dd0472c) after merge of Immutable related commit (https://github.com/bgrins/gecko-dev/pull/81)

File: devtools/client/webconsole/new-console-output/reducers/messages.js [hunk 1]
```diff
@@ -12,6 +12,11 @@ function messages(state = Immutable.List(), action) {
   switch (action.type) {
     case constants.MESSAGE_ADD:
       let newMessage = action.message;
+
+      if (newMessage.data.level === "clear") {
+        return Immutable.List(newMessage);
```
Comment (nchevobbe) [inline devtools/client/webconsole/new-console-output/reducers/messages.js]: fixed in the next commit (https://github.com/bgrins/gecko-dev/pull/82/commits/4e7bff19a4e3b5dcbf7e0f1fc100e1308dd0472c#diff-86d34eb522b93a83a28036670ed399d1R17)

**Comment by bgrins** [comment]
Nice!

**Comment by nchevobbe** [comment]
Redux made it quite easy :)
Any issue you'd prefer I work on now ?

**Comment by bgrins** [comment]
>  Any issue you'd prefer I work on now?

Thanks, yeah I'd say any of the other renderers, like console.trace(), console.count(), or input messages from jsterm.  console.count() (#67) might be a relatively easy one since it seems to just be a generic message with some error handling for the message.counter property

**Pull request merged**